src/client/Microsoft.Identity.Client/Platforms/Android/Broker/AndroidBroker.cs

```diff
@@ -54,10 +56,23 @@ public bool IsBrokerInstalledAndInvokable()
             }
         }
 
+        private void CheckPowerOptimizationStatus(bool CheckPowerOptimizations = true)
```

Comment:
>bool CheckPowerOptimizations = true [](start = 50, length = 35)

why is this value passed in if it's not used in the method? in the linked android code, they are also checking `skipPowerOptimizationCheck && connectionService.isNetworkDisabledFromOptimizations()`, but we aren't using the passed in value anywhere. does PowerManager not have a "isNetworkDisabledFromOptimizations" mode, as it seems we are looking for different things compared w/Android team.